TanitaTrackerNetCore/DatabaseMigration/Migrations/20180602204300_create_grant_table.cs

```diff
@@ -0,0 +1,19 @@
+﻿using DatabaseMigration.Framework;
+using Domain.Entities;
+using FluentMigrator;
+
+namespace DatabaseMigration.Migrations
+{
+    [Migration(20180602204300)]    
+    public class _20180602204300_create_grant_table : SoftDeleteEntityMigration<Grant>
+    {
+        public override void Up()
+        {
+            base.Up();
+            CreateForeignKeyColumn<Roles>(nameof(Entity.RoleId));
+            CreateForeignKeyColumn<Module>(nameof(Entity.ModuleId));
```

Comment:
Que representa el Module ?

Reply:
Eso vienen siendo los módulos. Ejemplo: Usuarios, clientes...

Reply:
Do a rebase with develop and I will merge it

Reply:
Ok